libs/designer/src/lib/ui/panel/templatePanel/createWorkflowPanel/tabs/reviewCreateTab.tsx

```diff
@@ -205,8 +181,8 @@ export const reviewCreateTab = (
         description: 'Button text for creating the workflow',
       })
     ),
-    primaryButtonOnClick: isCreated ? () => TemplateService()?.openBladeAfterCreate(workflowName) : onCreateClick,
-    primaryButtonDisabled: isPrimaryButtonDisabled || isCreating,
+    primaryButtonOnClick: onCreateClick,
+    primaryButtonDisabled: isPrimaryButtonDisabled || isCreating || isCreated,
     secondaryButtonText: isCreated
```

Comment:
If the secondaryButtonOnClick behavior is the same on "isCreated", then it should show the same text "Previous"

Reply:
We are auto closing the panel, so this should not matter, hence removed it